### DISCUSSION TITLE
Just wanted to say.. this is pretty amazing! [Apr 22]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1934] [Discord Threads]
> 🙋‍♂️ **jdmry**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### Thank you, this means a lot to our small team   
####    
#### Please help spread a word or two at your workplace and friends 😄  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/966717274448887819)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
